index.html

```diff
@@ -1,18 +1,92 @@
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+    <div class="container">
+      <nav>
+        <div>
+          <img src="images/spotify-logo.png" width="260px">
+        </div>
+  
+        <div class="boxes">
+          <a class="box">Premium</a>
+          <a class="box">Discover</a>
+          <a class="box">Help</a>
+          <a class="box">Download</a>
+        </div>
+      </nav>
+  
+      <section class="bigpicture">
+        <div>
+          <h1>Music for everyone.</h1>
+          <h3>Spotify is now free on mobile, tablet and computer. Listen to the
+            right music, wherever you are.</h3>
+        </div>
+      </section>
+  
+      <main>
+  
+        <div class="center">
+          <h2>What’s on Spotify?</h2>
+        </div>
+  
+        <div class="descriptions">
+          <article class="tocenter">
+            <img src="images/music-icon.png" height="150px">
+            <h3>Millions of Songs</h3>
+            <p>There are millions of songs on Spotify </p>
+          </article>
+  
+          <article class="tocenter">
+            <img src="images/high-quality-icon.png" height="150px">
+            <h3>HD Music</h3>
+            <p>Listen to music as if you were listening live</p>
+          </article>
+  
+          <article class="tocenter">
+            <img src="images/devices-icon.png" height="150px">
+            <h3>Stream Everywhere</h3>
+            <p> Stream music on your smartphone, tablet or computer</p>
+          </article>
+        </div>
+        
+      </main>
+  
+      <footer>
+  
+        <div class="footertext">
+          <h2>It’s as yeezy as Kanye West.</h2>
+          <article>
+            <h3>Search</h3>
+            <p>Know what you want to listen to? Just search and hit play.</p>
+          </article>
+          <article>
+            <h3>Browse</h3>
+            <p>Check out the latest charts, brand new releases and great playlists for right now.</p>
+          </article>
+          <article>
+            <h3>Discover</h3>
+            <p>Enjoy new music every Monday with your own personal playlist. Or sit back and enjoy Radio.</p>
+          </article>
+        </div>
+  
+        <div class="spotifylogo">
+          <img src="images/spotify-icon-white.png" width="150px">
+        </div>
+  
+        <div class="appimage">
+          <img src="images/spotify-app.jpg" height="650px">
```

Comment:
A thought and suggestion here. When using the attributes ```width``` or ```height``` on an element, it isn't necessary to give ```px``` to the values. This is because the ```width``` and ```height``` attributes already specified in pixels natively by HTML.

Additionally, when using ```<img>``` tags, it is important to also include the ```alt``` attribute. This is because in case the image isn't able to load, the ```alt``` value will be displayed to the user. The ```alt``` value is just a string that is needed. 

So in the case of all the ```<img>``` tags, you can remove the "px" and include the ```alt``` attribute. See the suggestion below:

```suggestion
          <img src="images/spotify-app.jpg" height="650" alt="spotify-app-image">
```